### PR TITLE
Clarification for P2SH in the developer guide

### PR DESCRIPTION
**Pull request by rod-lin**
Hi! I fixed the issue #1924. Does the bounty still count? :-)

My address is 3GsjjePXvMqoTYucCXgFbm4uRnt7NdAj59

### CODE REVIEW

File: _data/devdocs/en/guides/transactions.md [hunk 1]
```diff
@@ -367,6 +367,14 @@ Pubkey script: OP_HASH160 <Hash160(redeemScript)> OP_EQUAL
 Signature script: <sig> [sig] [sig...] <redeemScript>
 ~~~
 
+{% autocrossref %}
+
+This scirpt combination looks perfectly fine to old nodes as long as the script hash matches the redeem script. However, after the soft fork is activated, new nodes will perform a further verification for the redeem script. They will extract the redeem script from the signature script, decode it, and execute it with the remaining stack items(&lt;sig&gt; &#91;sig&#93; &#91;sig..&#93;part). Therefore, to redeem a P2SH transaction, the spender must provide the valid signature or answer in addition to the correct redeem script.
```
Comment (wbnns) [inline _data/devdocs/en/guides/transactions.md]: @rod-lin Could you please fix the typo? It looks like "script" was accidentally spelled as "scirpt" in the first sentence.

**Comment by rod-lin** [comment]
@wbnns 😂 Sorry about that... I fixed it.

**Comment by wbnns** [comment]
@rod-lin No worries!

Unless others object, this will be merged on Friday, June 15th.

**Comment by rod-lin** [comment]
👌 Thanks!

**Comment by wbnns** [comment]
@rod-lin 

Here's the [transaction ID](https://live.blockcypher.com/btc/tx/5bf58e175d1361a75b25be6518daed50e19c50cf537d77657b40c90ca199fef4/) for the bounty you took down.

**Pull request merged**